<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageDialog</class>
 <widget class="QewExtensibleDialog" name="ImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ImageDialog</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="cleanWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="cleanLabel">
          <property name="text">
           <string>Clean</string>
          </property>
          <property name="margin">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="cleanToolButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/plugin-resources.qrc">
            <normaloff>:/database/icons/eraser.png</normaloff>:/database/icons/eraser.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="pathWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="pathLabel">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathLineEdit">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pathToolButton">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nameToolButton">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="typeWidget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="typeLabel">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="typeComboBox">
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>70</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="descriptionTextEdit">
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>titleLabel</zorder>
  <zorder>pathWidget</zorder>
  <zorder>cleanWidget</zorder>
  <zorder>cleanLabel</zorder>
  <zorder>cleanLabel</zorder>
  <zorder>horizontalSpacer</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QewExtensibleDialog</class>
   <extends>QDialog</extends>
   <header>qewextensibledialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/plugin-resources.qrc"/>
 </resources>
 <connections/>
</ui>
